libraries/abstractions/transport/utest/CMakeLists.txt

```diff
@@ -58,8 +48,8 @@ set(real_name "${project_name}_real")
 create_mock_list(${mock_name}
             "${mock_list}"
             "${CMAKE_CURRENT_LIST_DIR}/project.yml"
-            "${mock_include_list}"
-            "${mock_define_list}"
+            ""
```

Comment:
These should remain as the section below line 43 (containing `==== (end edit) === `) represents the CMake configuration that is usually not edited